after/syntax/javascript.vim

```diff
@@ -19,7 +19,18 @@ hi! link jsDocTypeBrackets         DraculaCyan
 hi! link jsFuncArgOperator         Operator
 hi! link jsFunction                Keyword
 hi! link jsTemplateBraces          Special
+hi! link jsModuleAsterisk          DraculaPurpleItalic
+hi! link jsVariableDef             DraculaPurple
```

Comment:
Outside the scope of this PR. Please revert.

Reply:
Actually, those values were picked by me to match how it looked on VSCode. I'm ok whether we leave them or not.

Reply:
I'm the maintainer of the vscode repo. Variables _should_ be white by default. The reason why they appear to be purple in vscode is because vscode incorrectly treats `const` variables as true constants. There is an issue open in the repo to fix this.

The module asterisk thing is fine, but it's outside the scope of this particular PR. So I would just ask that it is opened in a different PR.

Reply:
Oh! Then I was trying to replicate a bug haha!... Thanks for spotting that out, and also thanks for your work on both repos then 🚀